api/tests/Feature/EventType/EventTypeApiTest.php

```diff
@@ -32,6 +32,7 @@ final class EventTypeApiTest extends TestCase
                 'end_date' => '2020-08-06 22:00:00',
             ]
         ],
+        'location_type' => 'zoom'
```

Comment:
Please add a test for the `address` type, and you will see that there are some errors in the code.